src/file/file.interface.ts

```diff
@@ -11,7 +11,6 @@ export interface IFile {
   type: string;
   description?: string;
   ownerID: string;
-  owner?: IUser;
```

Comment:
Remove unused import of `IUser` in [line 1](https://github.com/meateam/file-service/blob/a1f222c6d57d630ee113eb08d6e1261698666e7c/src/file/file.interface.ts#L1).